src/main/java/datawave/microservice/metadata/tablecache/TableCacheController.java

```diff
@@ -0,0 +1,78 @@
+package datawave.microservice.metadata.tablecache;
+
+import org.apache.log4j.Logger;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.codahale.metrics.annotation.Timed;
+
+import datawave.core.common.result.AccumuloTableCacheStatus;
+import datawave.webservice.result.VoidResponse;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@Tag(name = "Table Cache Controller /v1", description = "DataWave Table Cache Operations",
+                externalDocs = @ExternalDocumentation(description = "Query Executor Service Documentation",
+                                url = "https://github.com/NationalSecurityAgency/datawave-query-executor-service"))
+@RestController
+@RequestMapping(path = "/v1/AccumuloTableCache", produces = MediaType.APPLICATION_JSON_VALUE)
+@ConditionalOnProperty(name = "datawave.table.cache.enabled", havingValue = "true", matchIfMissing = true)
+public class TableCacheController {
+    private final Logger log = Logger.getLogger(TableCacheController.class);
+    private final TableCacheReloadService service;
+    
+    public TableCacheController(TableCacheReloadService service) {
+        this.service = service;
+    }
+    
+    /**
+     * <strong>JBossAdministrator or Administrator credentials required.</strong> Force a reload the table cache for the specified table.
+     *
+     * @return a VoidResponse
+     * @RequestHeader X-ProxiedEntitiesChain use when proxying request for user
+     * @RequestHeader X-ProxiedIssuersChain required when using X-ProxiedEntitiesChain, specify one issuer DN per subject DN listed in X-ProxiedEntitiesChain
+     * @HTTP 200 success
+     * @HTTP 500 internal server error
+     */
+    @Operation(summary = "Force a reload the table cache for the specified table.")
```

Comment:
This did not read correctly to me.. maybe I am misunderstanding. If my suggestion is correct, the javadoc will have to change as well.
```suggestion
    @Operation(summary = "Force a reload of the table cache for the specified table.")
```